src/api/actions/checkBulkUploadJob.ts

```diff
@@ -0,0 +1,21 @@
+import { Maybe } from "../../utilities";
+import { UploadJobStatus } from "../../types";
+import { makeApiCall } from "./helpers";
+
+export const checkBulkUploadJob = async (
+  apiUri: string,
+  urls: string[]
+): Promise<UploadJobStatus[]> => {
+  let response: Maybe<UploadJobStatus[]>;
+  try {
+    response = await makeApiCall<UploadJobStatus[]>(
+      `${apiUri}/checkBulk`,
+      "POST",
+      {urls: urls}
```

Comment:
This doesn't look like the correct input to the function

Reply:
fixed this as well